### PR TITLE
Add PowerPC prohject support

### PR DESCRIPTION
**Pull request by k0da**
Signed-off-by: Dinar Valeev <dvaleev@suse.com>

### CODE REVIEW

File: doit.sh [hunk 1]
```diff
@@ -9,7 +9,25 @@ else
...
 
-echo "generate dvds for $proj at "
+case $proj in
+	Factory)arches="i586 x86_64"i
+		repo="standard"
+		;;
+	Factory:PowerPC) arches="ppc64 ppc64le"
+		repo="standard"
+		;;
+	openSUSE:13.3) arches="ppc64 ppc64le"
```
Comment (coolo) [inline doit.sh]: I tend to belive that 13.3 will be release for more than that :)

File: commit.sh [hunk 2]
```diff
@@ -3,49 +3,52 @@
...
+   osc -q up -u osc/openSUSE:$proj:Live/package-lists-x11.* > /dev/null
+   for arch in $arches; do
+       cp -a output/opensuse/$proj/*default.$arch.list osc/openSUSE:$proj:Live/package-lists-images.$arch
+       cp -a output/opensuse/$proj/kde4_cd.$arch.list osc/openSUSE:$proj:Live/package-lists-kde.$arch/packagelist
+       cp -a output/opensuse/$proj/gnome_cd.$arch.list osc/openSUSE:$proj:Live/package-lists-gnome.$arch/packagelist
+       cp -a output/opensuse/$proj/x11_cd.$arch.list osc/openSUSE:$proj:Live/package-lists-x11.$arch/packagelist
+   done
+   osc -q ci -m "auto update" osc/openSUSE:$proj:Live/package-lists-* | grep -v nothing
+
+   installcheck x86_64 --withobsoletes trees/openSUSE:$proj-$repo-x86_64.solv > openSUSE:$proj.installcheck
```
Comment (coolo) [inline commit.sh]: the installcheck might be interesting for power too

Comment (k0da) [inline commit.sh]: I can add this later..

File: doit.sh (hunk 1 above)
Comment (k0da) [inline doit.sh]: Yeah.. I've fixed that.

**Pull request merged**